api/proto/teleport/assist/v1/assist.proto

```diff
@@ -113,6 +113,14 @@ message IsAssistEnabledResponse {
   bool enabled = 1;
 }
 
+// DeleteAssistantConversationRequest is a request to delete the conversation.
+message DeleteAssistantConversationRequest {
+  // conversationId is a unique conversation ID.
```

Comment:
I'm not sure about this but shouldn't this use snake case?

```suggestion
  // conversation_id is a unique conversation ID.
```

Reply:
We're using Go-ish case. Otherwise, the generated GoDocs have incorrect names :man_shrugging: